### ISSUE TITLE
Add help

### ISSUE THREAD

**Issue by xavierfav** [xavierfav/multi-web-audio-sequencer#49]
It would be nice to have instructions for showing to user how to user the sequencer, and what are the functionalities.

We though about some pop-up, or a kind of modal on top of the page that would appear the first time you come to the app. Also, a button could make it appear again.

A small brainstorming might be worth before starting to implement something.
So, what are the explanation to give? And where the pop-up should point?

Here are the different functionalities. Maybe it is not needed to explain them all:
- Rooms
- Nickname & chat
- Play/Pause
- Record & Exported audio
- Tempo
- Sequence length
- Name, drop down
- Start time/ end time
- Gain knob
- Delete
- Add track
- Freesound search
- Duration filter
